docs/release-notes/release-2023-07-11.md

```diff
@@ -0,0 +1,92 @@
+- Improve formatting for properties with comments and decorators
+- Support for new Unicode 15.0 identifier characters
+
+### Emitter framework
+
+- Add `writeOutput` to TypeEmitter for additional customization in writign generated files
+- Add `meta`` property to source files and declarations to store arbitrary metadata
+- Add support for emitting enum member references
+- Add new `TypeEmitter` methods for scalar instantiation
+
+### `@typespec/http` library
+
+- Add documentation on automatic header name resolution from property name for `@header` decorator
+
+### `@typespec/json-schema` emitter
+
+- Support `@extension` for adding arbitrary vendor extensions into the output
+- Add support for `Record<T>`
+- Support templates instantiated with intrinsic types and type expressions
+- Export emitter and related types from the npm package
+- By default, types that are not marked with @jsonSchema or are within a namespace with @jsonSchema are bundled into the schemas that reference them. Set the `emitAllRefs` option to true to get the previous behavior of emitting all types referenced as JSON Schema
+
+### `@typespec/openapi3` emitter
+
+- Omit `x-typespec-name` extension by default from openapi3 output. A new configuration and command line option `include-x-typespec-name: \"inline-only\" | \"never\"` has been added to get previous behavior
+
+### `@typespec/rest` library
+
+- Add `isListOperation` function migrated from `@typespec/compiler`
+- Make internal `@resourceTypeForKeyParam` decorator private
+
+## Bug Fixes
+
+### `@typespec/compiler` and core language bug fixes
+
+- Fix signature help after comment with no closing parenthesis or angle bracket
+- Doc comment `/** */` should override base type doc in `model is` or `op is`
+- Formatter incorrectly formatting `::` to `.`
+- Fix formatting issue with comment between decorator and scalar or interface
+- Stack overflow when having circular reference with `op is` inside an interface.
+- Clean up handling of non-standard entrypoints in language server
+- Fix stack overflow when referencing decorator target in checker and projections
+
+### `@typespec/openapi3` emitter bug fixes
+
+- OpenAPI emitters: treat union namespaces like namespaces for other types
+
+### Emitter framework bug fixes
+
+- Fix that some context methods were not being passed the expected parameters
+- Fix that context was set incorrectly for some `TypeEmitter` methods, and add missing context methods for model properties, enum members, and union variants
+
+### `@typespec/json-schema` emitter bug fixes
+
+- Json Schema Emitter: Fix a bug that could result in a schema being bundled more than once
+
+## Breaking Changes
+
+### For Specification Authors
+
+- Update tsp init template schema for future extensibility. Older tsp version will fail validation when trying to initialize a project with a new template definition.
+- Json Schema Emitter: the namespace has been corrected to TypeSpec.JsonSchema
```

Comment:
```suggestion
- Json Schema Emitter: the namespace has been corrected to TypeSpec.JsonSchema. Update any using statements from `JsonSchema` to `TypeSpec.JsonSchema`, and any references from `JsonSchema.[Type]` to `TypeSpec.JsonSchema.[Type]`.
```

Might be a bit too detailed but I feel it's always good to say how to fix an error if you can.